templates/themes/app/content/macros/video.html.twig

```diff
@@ -111,15 +124,15 @@
 
         {% if 'youtube' in video_types %}
             <div class="video-youtube ratio ratio-16x9">
-                <iframe frameborder="0" src="https://www.youtube.com/embed/{{ video_identifier }}{% if autoplay %}?autoplay=1{% endif %}" width="770" height="433" allowfullscreen></iframe>
+                <iframe frameborder="0" src="https://www.youtube.com/embed/{{ video_identifier }}{% if autoplay %}?autoplay=1{% endif %}" {% if video_title %}title="{{ video_title }}" {% endif %}width="770" height="433" allowfullscreen></iframe>
```

Comment:
I don't think we need `{% if video_title %}` since the `video_title` variable is always defined with the earlier change in the same file.